pyproject.toml

```diff
@@ -132,8 +138,13 @@ logo = "art/logo.png"
 palette = {scheme = "isort"}
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[dependency-groups]
+dev = [
```

Comment:
What is the difference between this and the `dev` optional dependencies?

Reply:
I did some mess in the conversion, but fixed.
[dependency-groups] are the correct part for dev dependencies, the optional dependencies are the extras.